tmail-backend/apps/distributed-es6-backport/pom.xml

```diff
@@ -414,6 +414,9 @@
                             <port>8000</port> <!-- WEBADMIN -->
                         </ports>
                         <appRoot>/root</appRoot>
+                        <jvmFlags>
+                            <jvmFlag>-Dextra.props=/root/conf/jvm.properties</jvmFlag>
```

Comment:
Thoughts: Do you think it's necessary? or do we add that in the helm instead directly?

Reply:
this can avoid bad surprises when running docker directly so imo lets keep it